contracts/Tokens/Prime/README.md

```diff
@@ -295,7 +305,9 @@ The OpenZeppelin Plausable contract is used. Only the `claimInterest` function i
 
 ## Calculate income per block
 
-First of all, we need to calculate the total protocol income per block. These are the formulas from @Kirill on how we can calculate this.
+_Note: This calculation is required only for estimation purposes and not for reward calculations. The real rewards are calculated as it is explained in the [Rewards](#rewards) section._
```

Comment:
_Note: This calculation should only be used to estimate the reward emissions. The exact rewards are calculated as it is explained in the [Rewards](#rewards) section._

Reply:
See [c72a86](https://github.com/VenusProtocol/venus-protocol/pull/358/commits/c72a8647cd6f359fecee2616c7d45206ff9c2cb2)